lang/py/avro/test/test_io.py

```diff
@@ -185,6 +186,23 @@ class DefaultValueTestCaseType(TypedDict):
             {"type": "record", "name": "ns.long", "fields": [{"name": "value", "type": "int"}, {"name": "next", "type": ["null", "ns.long"]}]},
             {"value": 0, "next": {"value": 1, "next": None}},
         ),
+        # Optional logical types.
+        (
+            [{"logicalType": "uuid", "type": "string"}, "null"],
+            None,
+        ),
+        (
+            [{"logicalType": "uuid", "type": "string"}, "null"],
+            uuid.uuid4().hex,
+        ),
+        (
+            [{"type": "long", "logicalType": "timestamp-millis"}, "null"],
+            datetime.datetime(1000, 1, 1, 0, 0, 0, 000000, tzinfo=avro.timezones.utc),
```

Comment:
```suggestion
            datetime.datetime(1000, 1, 1, 0, 0, 0, 0, tzinfo=avro.timezones.utc),
```
Just to make code scanning happy.